src/test/java/com/google/api/codegen/protoannotations/testdata/library_v2_gapic.yaml

```diff
@@ -673,13 +673,36 @@ interfaces:
                 - parameter: name%book_id
                   sample_argument_name: big_book_name
                   description: "The name of the book."
+          - id: test_long_running_start_then_cancel_calling_form
+            title: "kick off, retrieve name, and cancel"
+            description: "Kick off the LRO and cancel it."
+            parameters:
+              defaults:
+              - name%shelf_id=Novel
+              - name%book_id="War and Peace"
+            on_success:
+            - comment: [You'll want to save this for later to retrieve your completed operation.]
+            - print: ["Operation Name: %s", $resp.name]
```

Comment:
This works well for our current use-cases!

I wish there were a way to have the best of both worlds & both show how to cancel & show how to get the result and print things from it. But no great ideas are coming to mind right now. I'd love an `$op` and a `$resp`. But this is really, really, really useful and unblocks us from generating and _running_ samples for LROs which take forever and/or cost a lot to run if allowed to run for awhile.